migrations/migrations/20180622182009-create-user.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
```

Comment:
Usually the attributes of a table are written in snake_case instead of camelCase.